src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -100,6 +135,11 @@ function NameAndPicture({ navigation }: Props) {
       return
     }
 
+    try {
+      Statsig.logEvent(StatsigEvents.ONBOARDING_NAME_STEP_COMPLETE)
```

Comment:
personally I'd prefer to use our existing analytics events and just publish them to StatSig via Segment, rather than have a bespoke parallel analytics event for everything we want to measure with StatSig

Reply:
Made an [issue](https://linear.app/valora/issue/ACT-467/segmentstatsig-integration) to handle this in a separate PR, assuming we decide to stick with Statsig